python/az/aro/azext_aro/_params.py

```diff
@@ -138,12 +138,12 @@ def load_arguments(self, _):
                    options_list=['--enable-managed-identity', '--enable-mi'],
                    validator=validate_enable_managed_identity)
         c.argument('platform_workload_identities', arg_group='Identity', is_preview=True,
-                   help='Assign a platform workload identity used within the cluster',
+                   help='Assign a platform workload identity used within the cluster. Requires two values: the use inside the cluster and the user assigned identity.',
```

Comment:
```suggestion
                   help='Assign a platform workload identity used within the cluster. Requires two values: the user inside the cluster and the user assigned identity.',
```

Just for clarification, is the user inside the cluster a ServiceAccount or a User? Same for the Azure side; what's the Azure terminology for the object?